cfeScraping/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const puppeteer = require('puppeteer');
+const URL = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+
+let OpenBrowser = async () => {
+    try {
+        const browser = await puppeteer.launch();
+        const page = await browser.newPage();
+
+        await page.goto(URL);
+        await page.waitForSelector('div.gti-app');
+        console.log('Successful');
+
+        await page.waitForTimeout(2000);
+        await page.screenshot({ path: 'pictures/Cfe.png' });
+
+        await browser.close();
+    } catch (error) {
+        console.error('An error have been occurred: ' + error);
```

Comment:
if something wrong happened (check the previous review) the browser is still opened & the program never ends.

![Capture](https://user-images.githubusercontent.com/5388080/126830608-9b27cf71-4945-456f-9d3d-d513031ed923.PNG)

What do you propose?